### PR TITLE
Reorganize Concepts in View Transitions spec

### PR DESCRIPTION
**Pull request by fantasai**
This patch splits apart the Concepts section which was a collection of random ideas and consolidates the various parts with related content.

It might be easier to review commit by commit.

### CODE REVIEW

File: css-view-transitions-1/Overview.bs [hunk 1]
```diff
@@ -499,8 +499,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	</pre>
 
 	The 'view-transition-name' property “tags” an element
-	as potentially [=involved in a view transition=],
-	capturing it independently in the [=view transition tree=]
+	for [=capture in a view transition=],
```
[the remark this answers is not in the document]
Comment (fantasai) [reply css-view-transitions-1/Overview.bs:502]: Actually, modern Bikeshed is smart enough to conjugate this, so they both link up correctly when built. :)

[the remark this answers is not in the document]
Comment (khushalsagar) [reply css-view-transitions-1/Overview.bs:502]: Great, TIL!

File: css-view-transitions-1/Overview.bs [hunk 2]
```diff
@@ -744,6 +727,69 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
...
+	with the following characteristics:
+
+	- Its parent [=stacking context=] is the root stacking context.
+
+	- If the ''::view-transition'' pseudo-element exists,
+		a new [=stacking context=] is created for the [=document element=] and the [=Document/top layer=].
+		The [=view transition layer=] is a sibling of this stacking context.
+
+	- The [=view transition layer=] paints after the stacking context for the [=document element=] and [=Document/top layer=].
+		This includes the filters and effects that are applied to the [=document element=].
```
[the remark this answers is not in the document]
Comment (fantasai) [reply css-view-transitions-1/Overview.bs:786]: I rewrote this part in https://github.com/w3c/csswg-drafts/pull/8927 so I think we should leave it untouched here. :)

File: css-view-transitions-1/Overview.bs [hunk 3]
```diff
@@ -1145,6 +1053,93 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
...
+
+			When this is true, [=this=]'s [=active view transition=]'s [=ViewTransition/transition root pseudo-element=] renders as a child of [=this=]'s [=document element=],
+			and [=this=]'s [=document element=] is its [=originating element=].
+
+			Note: The position of the [=ViewTransition/transition root pseudo-element=] within the [=document element=] does not matter, as the [=ViewTransition/transition root pseudo-element=]'s [=containing block=] is the [=snapshot containing block=].
+	</dl>
+
+### Additions to Elements ### {#elements-concept}
+
+	[=/Elements=] have an <dfn export>captured in a view transition</dfn> boolean, initially false.
```
[the remark this answers is not in the document]
Comment (fantasai) [reply css-view-transitions-1/Overview.bs:1092]: Fixed, thanks!

**Pull request merged**